src/server/game/Entities/Player/Player.cpp

```diff
@@ -4874,117 +4874,117 @@ void Player::DurabilityPointLossForEquipSlot(EquipmentSlots slot)
         DurabilityPointsLoss(pItem, 1);
 }
 
-uint32 Player::DurabilityRepairAll(bool cost, float discountMod, bool guildBank)
+void Player::DurabilityRepairAll(bool takeCost, float discountMod, bool guildBank)
 {
-    uint32 TotalCost = 0;
-    // equipped, backpack, bags itself
-    for (uint8 i = EQUIPMENT_SLOT_START; i < INVENTORY_SLOT_ITEM_END; i++)
-        TotalCost += DurabilityRepair(((INVENTORY_SLOT_BAG_0 << 8) | i), cost, discountMod, guildBank);
+    std::list<std::pair<Item*, uint32>> itemRepairCostStore;
+    // Сollecting all items that can be repaired and repair costs
+    {
```

Comment:
is this an additional local scope ?

Reply:
> is this an additional local scope ?

Yes, just to highlight this separate piece of code. Technically, it is not necessary to do this.

Reply:
We usually don't use scopes to group code in a single function but mostly only to trigger a destructor. Could you please remove the additional scope in this case :) ?